site/.vitepress/config.ts

```diff
@@ -641,11 +624,8 @@ export default defineConfig({
             text: "Utils",
             collapsed: true,
             base: "/packages/aa-ethers/utils",
+            link: "/introduction",
             items: [
-              {
-                text: "Overview of Utils",
-                link: "/introduction",
-              },
```

Comment:
let's put this back, per my comment re: no section header pages to make each page explicitly visible on sidebar